packages/tldraw/src/lib/shapes/text/toolStates/Pointing.ts

```diff
@@ -15,29 +15,50 @@ export class Pointing extends StateNode {
 
 	markId = ''
 
+	enterTime = 0
+	override onEnter(): void {
+		this.enterTime = Date.now()
+	}
+
 	override onExit() {
 		this.editor.setHintingShapes([])
 	}
 
 	override onPointerMove(info: TLPointerEventInfo) {
+		// Create a fixed width shape if the user wants to do that.
+
 		// Don't create a fixed width shape unless the the drag is a little larger,
 		// otherwise you get a vertical column of single characters if you accidentally
 		// drag a bit unintentionally.
+
+		// If the user hasn't been pointing for more than 150ms, don't create a fixed width shape
+		if (Date.now() - this.enterTime < 150) return
+
 		const { editor } = this
-		const { isPointing, originPagePoint, currentPagePoint } = editor.inputs
-		if (
-			isPointing &&
-			Math.abs(originPagePoint.x - currentPagePoint.x) ** 2 >
-				((editor.getInstanceState().isCoarsePointer
-					? editor.options.coarseDragDistanceSquared
-					: editor.options.dragDistanceSquared) *
-					4) / // double the necessary drag distance for text shapes
-					editor.getZoomLevel()
-		) {
+		const { isPointing } = editor.inputs
+
+		if (!isPointing) return
+
+		const { originPagePoint, currentPagePoint } = editor.inputs
+
+		const currentDragDist = Math.abs(originPagePoint.x - currentPagePoint.x)
+
+		const baseMinDragDistForFixedWidth = Math.sqrt(
+			editor.getInstanceState().isCoarsePointer
+				? editor.options.coarseDragDistanceSquared
+				: editor.options.dragDistanceSquared
```

Comment:
This distance is squared by default (to skip sqrts in regular mouse move checks) so here we need to sqrt it back into regular units.